### DISCUSSION TITLE
locked out of budibase admin

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#8689] [Help]
I have budibase hosted on digitalocean and mistakenly changed my admin user to power user using the rest API.  Now I am locked out of all my apps.  Is there anyway to either change the user back to admin from the command line or export the apps also from the command line?  I thought of accessing users' table on couchdb but it seems I am connecting to the wrong db.  The right one is inside the budibase container under /data/couch/dbs/.  I keep accessing the db in the couchdb-service.  Appreciate some help.

**Comment by ihabmzaky** [comment]
![image](https://user-images.githubusercontent.com/109164523/201963443-e53d7ce5-bb1a-47c4-9bc3-e60eb9e8d71b.png)

**Comment by melohagan** [comment]
Hey @ihabmzaky 

That's a pickle! If you log-in to couchdb (you may need to expose the port, there's a tutorial on how to do that in DigitalOcean around the 2 min mark: https://youtu.be/p3bBSf0QtrE)

The credentials can be found in the `.env` file. 

Next go to the **global-db**. You'll be able to find entries that start with *us_*. These are user rows. One of them will be the admin account which you should be able to manually edit the role back to admin.

<img width="709" alt="Screenshot 2022-11-21 at 11 01 09" src="https://user-images.githubusercontent.com/101575380/203034232-f9b0571a-7480-4d06-89a7-85c226fdd648.png">

**Comment by ihabmzaky** [comment]
thank you.  will give it a try
